.github/workflows/build.yml

```diff
@@ -121,10 +122,10 @@ jobs:
         uses: actions/setup-java@v2
         with:
           distribution: 'adopt'
-          java-version: 11
+          java-version: 8.0.242
```

Comment:
This makes the compile fail.  I'll change to try setting up 8.0.242, then 11, then use a custom toolchain location to point at 8.0.242

https://docs.gradle.org/current/userguide/toolchains.html